packages/launch/src/utils.ts

```diff
@@ -25,14 +25,6 @@ export type PlanProduct = {
 	};
 };
 
-export const getPlanProduct = ( plan: Plans.Plan, flow: string ): PlanProduct => ( {
-	product_id: plan.productId,
-	product_slug: plan.storeSlug,
-	extra: {
-		source: flow,
-	},
```

Comment:
Let's keep this in some form as mentioned in https://github.com/Automattic/wp-calypso/pull/48790/files#r558595123